dist/profile/manifests/datadog_ssl_check.pp

```diff
@@ -0,0 +1,29 @@
+#
+# Define datadog check for ssl expiration
+# Inspired by: https://workshop.avatarnewyork.com/project/datadog-ssl-expires-check/
+#
+
+class profile::datadog_ssl_check (
+  $sites = undef,
```

Comment:
Instead of `undef`, I recommend making the default an empty array here, that way there won't be a Puppet error then the ERB template below is evaluated (`sites.each`)